release/release_logs/1.10.0/stress_tests/stress_test_many_tasks.json

```diff
@@ -0,0 +1,19 @@
+{
+  "success": 1,
+  "stage_0_time": 5.358870506286621,
+  "stage_1_time": 216.38800859451294,
+  "stage_1_avg_iteration_time": 21.638786935806273,
+  "stage_1_max_iteration_time": 22.329787492752075,
+  "stage_1_min_iteration_time": 21.30394411087036,
+  "stage_2_time": 251.29053378105164,
+  "stage_2_avg_iteration_time": 50.257874202728274,
+  "stage_2_max_iteration_time": 51.48736071586609,
+  "stage_2_min_iteration_time": 47.86618733406067,
+  "stage_3_creation_time": 0.06580400466918945,
+  "stage_3_time": 2932.907886981964,
```

Comment:
Hmm this has 50% regression (`  "stage_3_creation_time": 0.05767488479614258,
  "stage_3_time": 1949.1530861854553,` -> this is 1.9). cc @ericl can you sign off if this is okay? 

Looks like the actor creation performance has regressed. cc @iycheng do you know any known issue? 

```
# Create a bunch of actors.
def stage3(total_num_remote_cpus, smoke=False):
    start_time = time.time()
    logger.info("Creating %s actors.", total_num_remote_cpus)
    actors = [Actor.remote() for _ in range(total_num_remote_cpus)]
    stage_3_creation_time = time.time() - start_time
    logger.info("Finished stage 3 actor creation in %s seconds.",
                stage_3_creation_time)

    num_tasks = 1000

    if smoke:
        num_tasks //= 25

    # Submit a bunch of small tasks to each actor. (approximately 1070 seconds)
    start_time = time.time()
    logger.info("Submitting many small actor tasks.")
    for N in [num_tasks, num_tasks * 100]:
        x_ids = []
        for i in range(N):
            x_ids = [a.method.remote(0) for a in actors]
            if i % 100 == 0:
                logger.info("Submitted {}".format(i * len(actors)))
        ray.get(x_ids)
    return time.time() - start_time, stage_3_creation_time
```

Reply:
So, "creation_time" is not regressed, but this just means "submission time" actually. It is time it takes to submit X number of actors asynchronously (so has no meaning).

Seems like the time X number of actors are created + submitting 1000 tasks to them has regression

Reply:
This looks like a bad regression in my mind. I think we need to at least identify the root cause.

Reply:
I agree with this (especially we have a critical user who cares this metrics). Maybe we should bisect? I can create a release blocker issue. We can also see if other critical tests have regressed (e.g., many node actor test)

Reply:
cc @ericl @scv119 let us know your thoughts.

Reply:
agree this is a release blocker.